### DISCUSSION TITLE
DSheetPilling: Check if Sheetpile becomes unstable

### DISCUSSION THREAD

**Discussion by MaartenBW** [Deltares/GEOLib#156] [General]
Hi all,

What is the best way to check if a calculation failed because the 1 or more stages are unstable. When this happens in a batch calculation, a SHD is still created that can be parsed by the output parser, but the verification results are missing. Also, when switching between verification method A and B, the behaviour is different. 

I think it would be nice of there is a flag in the OutputStructure to indicate if the calculation did succeed, or do i miss something?

Thanks,

Maarten

**Comment by wfaustmann** [comment]
The identifier AllStagesAreStable in the [MAXIMUM SUMMARY RESULTS]-block indicates whether one of more stages are unstable.
To have more insight on the specific stage that is unstable, the CalculationStatus in the [RESUME]-block can be analysed.
